.github/workflows/create_example_nbs.yml

```diff
@@ -0,0 +1,39 @@
+name: "Convert Examples to Notebooks"
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "examples/pytorch/**.py"
+  pull_request:
+    branches: [main]
+    paths:
+      - "examples/pytorch/**.py"
+  workflow_dispatch:
+
+jobs:
+  convert-to-nbs:
+    name: "Convert examples to notebooks"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+      - name: Set Up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.7
+      - name: Set Up Environment
+        run: |
+          make install-uv reset-venv
+          source .venv/bin/activate
+          uv pip install nbformat jupytext
+      - name: Convert using Script
+        run: |
+          make generate-example-notebooks
+      - name: Check for diff
+        run: |
+          git add examples/notebooks/
+          if ! git diff --cached --exit-code; then
+            echo "Notebooks have changed! Please commit the changes."
+            exit 1
```

Comment:
Did you test if the action fails if a notebook was not updated?

Reply:
Just did, it failed here https://github.com/lightly-ai/lightly/actions/runs/10318711916/job/28565632396?pr=1627

Reply:
Yes, this run fails here [04b5bec](https://github.com/lightly-ai/lightly/pull/1627/commits/04b5beca56ca2e56801b70aabb764bab51582f12)

Reply:
And succeeds here https://github.com/lightly-ai/lightly/actions/runs/10318986807

Reply:
Are you sure this works correctly, the failing run seems to be failing because of a jupytext import error and not the git diff: https://github.com/lightly-ai/lightly/actions/runs/10318945569/job/28566351761